src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -670,7 +680,7 @@ export class ServerView extends DirMixin(PolymerElement) {
       this.selectedTab = 'connections';
       this.accessKeySortBy = 'name';
       /** The direction to sort: 1 == ascending, -1 == descending */
-      this.accessKeySortDirection = 1;
+      this.accessKeySortDirection = 0;
```

Comment:
Maybe we keep the ascending sort by default? This way it would keep the unnamed access keys at the bottom.

I'm wondering if users will get confused when they upgrade and see the different sort. Although either way the order of the keys will be different :/

Reply:
The no-sorting will display by creation order, so the new, unnamed keys will stay at the bottom.
The unnamed keys are unaffected by sorting by name.

Reply:
@smithmonique Should the keys be sorted by creation time or name by default?

"Unsorted" (creation time) feels more natural, but I can see how making it sorted by name increases the visibility of the sorting feature, since it would show an arrow on the column header.